src/components/ebay-confirm-dialog/test/test.browser.js

```diff
@@ -0,0 +1,66 @@
+const assign = require('core-js-pure/features/object/assign');
+const { expect, use } = require('chai');
+const { render, fireEvent, wait, cleanup } = require('@marko/testing-library');
+const { fastAnimations } = require('../../../common/test-utils/browser');
+const template = require('..');
+const mock = require('./mock');
+
+use(require('chai-dom'));
+before(fastAnimations.start);
+after(fastAnimations.stop);
+afterEach(cleanup);
+
+/** @type import("@marko/testing-library").RenderResult */
+let component;
+
+describe('given a closed dialog', () => {
+    const input = mock.Dialog;
+
+    beforeEach(async() => {
+        component = await render(template, input);
+    });
+
+    it('then it is hidden in the DOM', () => {
```

Comment:
Do we repeat these same tests for every dialog type? Can't we just test once for the base logic?

Reply:
We do test the base logic. But need to check this one is working too. Plus it increases coverage since this dialog wont work